docs/pages/refactoring.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Refactoring tools
+layout: default
+---
+
+# Refactoring Tools
+
+| Name                                                        | Short description                                              | 🚦  |
+| ----------------------------------------------------------- | -------------------------------------------------------------- | --- |
+| [jedi](https://jedi.readthedocs.io/en/stable/)              | Refactoring and autocompletion used in IDEs.                   | 🟢  |
```

Comment:
With my undefined-acronym hunter 🎩  on - maybe worth expanding IDEs to _integrated development environments_? Depends whether we think all potential users will know what an IDE is. It might be that IDE in short form is actually more well recognized than the full _integrated development environment_ thought.

Reply:
Agreed. How about linking the acronym to [the Wikipedia article](https://en.wikipedia.org/wiki/Integrated_development_environment)?

Reply:
Yep that seems a good solution